src/modelValidator.js

```diff
@@ -1,357 +1,469 @@
 import * as jsonc from 'jsonc-parser'
+import {filter, groupBy, map, mergeMap, toArray, flatMap, distinct } from 'rxjs/operators';
+import {from} from 'rxjs';
+import * as lodash from 'lodash'
```

Comment:
Where do we use lodash? We don't have it as an explicit dependency, as far as I know